src/libstd/sys/wasm/time.rs

```diff
@@ -51,6 +51,10 @@ impl SystemTime {
         SystemTime(self.0 + *other)
     }
 
+    pub fn checked_add_duration(&self, other: &Duration) -> Option<SystemTime> {
+        self.0.checked_add(other).map(|d| SystemTime(d))
```

Comment:
```suggestion
        self.0.checked_add(*other).map(|d| SystemTime(d))
```

Reply:
Thank you for the apply-ready suggestion, but I edited the faulty commit (and fixed another typo that I found while looking at the other implementations).